cfgmgr/buffermgrdyn.cpp

```diff
@@ -2932,6 +2932,12 @@ task_process_status BufferMgrDynamic::handleSingleBufferPgEntry(const string &ke
     buffer_pg_t &bufferPg = m_portPgLookup[port][key];
     port_info_t &portInfo = m_portInfoLookup[port];
 
+    if (portInfo.cable_length == "0m")
+    {
+        SWSS_LOG_NOTICE("Skipping profile creation for cable length '0m'");
```

Comment:
If I configure it to 0m and then add a PG, the PG configuration is lost.
When you change the cable length to other values, no profile will be created

Reply:
OK understand, I'll move it to allocateProfile() API instead.